config/locales/en/drafting_requirements.yml

```diff
@@ -0,0 +1,7 @@
+en:
```

Comment:
We can push this into the edit.yml for documents as this copy will only appear on this page.

Reply:
sure